### ISSUE TITLE
Minor corrections to Ex5.10, S9.5 & App F

### ISSUE THREAD

**Issue by RosalynHatcher** [cf-convention/cf-conventions#186]
A few inconsistencies:

Example 5.10:
- missing `;` on lines 8 & 12

Section 9.5:
- `cf_role` attribute is only allowed on coordinate variables by table in Appendix A. I think paragraph 2 in 9.5 should say _"Where feasible a coordinate or auxiliary coordinate variable with the attribute cf_role should be included."_

Appendix F:
- Table 1: Attribute `grid_mapping_name` should be type 'S' not Numeric.

Happy to create a PR if people agree with this.
Cheers,
Ros.

**Comment by dblodgett-usgs** [comment]
Example 5.10: http://cfconventions.org/cf-conventions/cf-conventions.html#british-national-grid
Looks good.

Section 9.5: http://cfconventions.org/cf-conventions/cf-conventions.html#coordinates-metadata
I'm not sure `cf_role` would ever be on what we might call an `auxiliary coordinate variable`?

Table F.1 doesn't have an appendix entry...
It is just above: http://cfconventions.org/cf-conventions/cf-conventions.html#_revision_history
Looks good.

**Comment by JimBiardCics** [comment]
@dblodgett-usgs If the variable is not named with its dimension it is, by definition, an auxiliary coordinate variable, even if it is 1D. This is also true if it is not monotonic. Example H.2 is a case in point. The station_name variable is not a coordinate variable. I think @RosalynHatcher's proposal is good.

**Comment by dblodgett-usgs** [comment]
👍 Got it. That makes sense. @RosalynHatcher -- want to open a PR?

**Comment by RosalynHatcher** [comment]
@dblodgett-usgs - Yes I'll do a PR

**Comment by evertrol** [comment]
Example 5.10 also has

```
float pres(z, y, x) ;
      temp:standard_name = "air_pressure" ;
      temp:units = "Pa" ;
      temp:coordinates = "lat lon" ;
      temp:grid_mapping = "crsOSGB: x y crsWGS84: lat lon" ;
```

That probably should be `pres:...` (looks like a copy-paste error).
It doesn't result in an error when using e.g. `ncgen`, but attributes for `pres` are lost, and those for `temp` will be overriden with the incorrect values.

Similarly, example 5.6 has 
```
float rlat(rlat) ;
    rlat:long_name = "latitude in rotated pole grid" ;
    rlat:units = "degrees" ;
    rlon:standard_name = "grid_latitude";
```
I assume the last line should be `rlat:standard_name = "grid_latitude";` instead?

Since the PR is still open, I hope this can be corrected in addition to the other fixes?
Otherwise I'll put in a separate PR (or someone else can).

**Comment by dblodgett-usgs** [comment]
@RosalynHatcher -- do you want to add these fixes to your open PR?

**Comment by RosalynHatcher** [comment]
@dblodgett-usgs:  Yes I will do.

**Comment by dblodgett-usgs** [comment]
Closed with #189
